cypress/support/fragments/data_import/logs/logsViewAll.js

```diff
@@ -132,7 +134,7 @@ export default {
       cy.get('[data-row-index]')
         .should('have.length', count)
         .then(() => {
-          this.getMultiColumnListCellsValues(2).then(statuses => {
+          this.getMultiColumnListCellsValues(3).then(statuses => {
```

Comment:
use start index + shift  instead of set of integers

Reply:
it's fixed in Javokhir pr